### ISSUE TITLE
reduce verbosity of cucumber tests

### ISSUE THREAD

**Issue by einhverfr** [ledgersmb/LedgerSMB#1856]
right now it is hard to follow testing progress in the test output.

We should, if possible, reduce the verbosity on the ccucumber test output (if we can)

**Comment by ehuelsmann** [comment]
Well, we can't, but upstream can. (meaning: I can.)

I need to find a day to write the weasel logging I've been planning to write. When I finish that, I'll remove the debug output too.

**Comment by einhverfr** [comment]
Since the logging is probably all debug logging, a simple solution might be to allow the user to pass in a logging function which would then handle the debug output, and the default one could just do nothing.

The nice thing about that approach is you have no responsibility but offer nearly infinite flexiility ;-)

**Comment by ehuelsmann** [comment]
Weasel now doesn't print any output anymore. The feature to implement a logger is still in Weasel and will be handled there.
